src/main/java/com/autotune/dbactivites/livenessProbe/TestDBConnection.java

```diff
@@ -0,0 +1,37 @@
+/*******************************************************************************
+ * Copyright (c) 2023 Red Hat, IBM Corporation and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *******************************************************************************/
+package com.autotune.dbactivites.livenessProbe;
+
+import com.autotune.dbactivites.init.KruizeHibernateUtil;
+import org.hibernate.Session;
+import org.hibernate.SessionFactory;
+
+import java.util.List;
+
+public class TestDBConnection {
+    public static void main(String[] args) {
+        System.out.println("Checking DB connection...");
+        SessionFactory factory = KruizeHibernateUtil.getSessionFactory();
+        Session session = factory.openSession();
```

Comment:
Shouldn't we add a null check here ? Should this code be within try/catch ?

Reply:
The purpose of this script is solely to facilitate the Liviliness probe, which aims to detect system errors that may cause the Kruize pod to fail to start if there is a problem with the database connection. Therefore, the script intentionally avoids using a try-catch mechanism.